sources/packages/web/src/views/supporting-user/SupportingInformation.vue

```diff
@@ -1,15 +1,14 @@
 <template>
   <full-page-container>
+    <body-header
+      title="Search for application"
+      subTitle="To provide your supporting information, please search for the application
+      by entering the requested information below. All fields are mandatory and
+      must match exactly with the information provided on the student
+      application."
+    >
+    </body-header>
```

Comment:
The search area for this form also needs some refactoring (IMO) to be in sync with "Search Institutions" and "Search Students". The validation right now is different and the date is not in the expected format. I would say that we need to:
1. Do the validations using the `v-form`;
2. Use the shared `useRules` composable for the rules;
3. Adjust the date format, at least for me it is rendered as below.
![image](https://user-images.githubusercontent.com/61259237/197830758-0142e6e3-b492-443f-843c-e1b17632a63d.png)

Since these were not captured in the previous PR reviews I would suggest wrapping up this PR and working in these changes right after.

Reply:
Ok. I'm gonna do that in another PR. Thanks!